packages/tupaia-web-server/src/routes/DashboardsRoute.ts

```diff
@@ -27,8 +27,42 @@ export type DashboardsRequest = Request<
 >;
 
 const NO_DATA_AT_LEVEL_DASHBOARD_ITEM_CODE = 'no_data_at_level';
+const NO_ACCESS_DASHBOARD_ITEM_CODE = 'no_access';
 
 export class DashboardsRoute extends Route<DashboardsRequest> {
+  private getNoDataDashboard = async (
+    rootEntityCode: Entity['code'],
+    staticDashboardItemCode: string,
+  ) => {
+    const { models } = this.req;
+    const noDataItem = await models.dashboardItem.findOne({
+      code: staticDashboardItemCode,
+    });
+
+    const { code, legacy, report_code: reportCode, id, config } = noDataItem;
+    return camelcaseKeys(
```

Comment:
No need for camelcaseKeys here as it is used to automatically convert objects from snake case to camelcase and you have already written out all the keys in camel case